dlio_benchmark/utils/utility.py

```diff
@@ -51,12 +56,52 @@ def utcnow(format=LOG_TS_FORMAT):
     return datetime.now().strftime(format)
 
 
-def get_rank():
-    return MPI.COMM_WORLD.rank
-
-
-def get_size():
-    return MPI.COMM_WORLD.size
+class mpi:
```

Comment:
Change name to DLIOMPI so that we don’t conflict classes with others and maintain correct formatting

Reply:
Yes, this it better.